conf-dist.ini

```diff
@@ -63,6 +63,7 @@ address =
 ;Dict of parameters used as configuration
 ca-parameters =
 ignore_capture_devices = False
+min-length = 0 ; Minimum duration for ingest in seconds: 0 indicates no limitiation (obs: 10)
```

Comment:
Clearer comment:

min-length = 0 ; Required minimum duration in seconds for a recording to be ingested